### PR TITLE
add data conversion based on data type

### PR DESCRIPTION
**Pull request by lcw2**
before the first byte is only taken.
Add support data conversion according to the number of bytes and data type.

### CODE REVIEW

**Comment by sailorvii** [comment]
@kevin-wangzefeng @fisherxu, please help to approve the running workflow. 
Thanks.

**Comment by fisherxu** [comment]
@lcw2  Please fix the failed ci tests.

**Comment by sailorvii** [comment]
/approve

Thanks for the improvement.

Please rebase the commits into 1.

File: mappers/modbus/device/twindata.go [hunk 1]
```diff
@@ -17,6 +17,10 @@ limitations under the License.
 package device
 
 import (
+	"encoding/binary"
+	"errors"
+	"github.com/kubeedge/mappers-go/mappers/modbus/configmap"
```
Comment (fisherxu) [inline mappers/modbus/device/twindata.go:22]: Move to kubeedge part.

**Comment by fisherxu** [comment]
@lcw2  Please also rebase to master branch and signoff the commit to fix the failed ci tests.

**Comment by lcw2** [comment]
ok

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**